src/composables/draw/draw-tooltip.ts

```diff
@@ -69,7 +68,7 @@ class DrawTooltip {
       const geom = geometry as LineString
       coord = geom.getLastCoordinate()
       if (coord !== null) {
-        output = getFormattedLength(geom, proj)
+        output = formatLength(getLength(geom, proj))
```

Comment:
I don't know if this is good practice to use a function coming from a directive (I would have suggest to move the formatLength in a service or helper or util and use it in both the directive and here). But it works great, so we can keep it as is.

Reply:
Also a good point, especially as we already have `formatting.utils`. I've moved the functions there.